src/components/mods.tsx

```diff
@@ -379,6 +379,10 @@ class AvailableModBox extends ModBox<IAvailableModState>
                             this.setState({installed: true});
                             }}>{installText}
                         </button>
+                        <div className="downloads">
```

Comment:
Would be nice to have this on installed mods too

Reply:
Added